packages/paste-core/components/table/stories/index.stories.tsx

```diff
@@ -0,0 +1,364 @@
+        </TFoot>
+      </Table>
+    );
+  })
+  .add('Column width', () => {
+    return (
+      <Table>
+        <THead>
+          <Tr>
+            <Th width="size50">&nbsp;</Th>
+            <Th>Column 1</Th>
+            <Th>Column 2</Th>
+            <Th>Column 3</Th>
+            <Th textAlign="right">Column 4</Th>
+          </Tr>
+        </THead>
+        <TBody>
+          <Tr>
+            <Td>1</Td>
+            <Td>Empty</Td>
+            <Td>Cell</Td>
+            <Td>Content</Td>
+            <Td textAlign="right" width="size40">
+              1
+            </Td>
+          </Tr>
+          <Tr>
+            <Td>2</Td>
+            <Td>Empty</Td>
+            <Td>Cell</Td>
+            <Td>Content</Td>
+            <Td textAlign="right">1</Td>
+          </Tr>
+          <Tr>
+            <Td>3</Td>
+            <Td>Empty</Td>
+            <Td>Cell</Td>
+            <Td>Content</Td>
+            <Td textAlign="right">1</Td>
+          </Tr>
+        </TBody>
+        <TFoot>
+          <Tr>
+            <Td>100</Td>
+            <Td>end</Td>
+            <Td>end</Td>
+            <Td>end</Td>
+            <Td textAlign="right">100</Td>
+          </Tr>
+        </TFoot>
+      </Table>
+    );
+  })
+  .add('Column textAlign', () => {
+    return (
+      <Table>
+        <THead>
+          <Tr>
+            <Th>&nbsp;</Th>
+            <Th>Left Aligned Column</Th>
+            <Th textAlign="center">Center Aligned Column</Th>
+            <Th textAlign="right">Right Aligned Column</Th>
+          </Tr>
+        </THead>
+        <TBody>
+          <Tr>
+            <Td>
+              <Text as="p" fontWeight="fontWeightBold">
+                First line of text
+              </Text>
+              <Text as="p">Second line of longer text text</Text>
+            </Td>
+            <Td>Cell</Td>
+            <Td textAlign="center">Content</Td>
+            <Td textAlign="right">1</Td>
+          </Tr>
+          <Tr>
+            <Td>
+              <Text as="p" fontWeight="fontWeightBold">
+                First line of text
+              </Text>
+              <Text as="p">Second line of longer text text</Text>
+            </Td>
+            <Td>Cell</Td>
+            <Td textAlign="center">Content</Td>
+            <Td textAlign="right">1</Td>
+          </Tr>
+          <Tr>
+            <Td>
+              <Text as="p" fontWeight="fontWeightBold">
+                First line of text
+              </Text>
+              <Text as="p">Second line of longer text text</Text>
+            </Td>
+            <Td>Cell</Td>
+            <Td textAlign="center">Content</Td>
+            <Td textAlign="right">1</Td>
+          </Tr>
+        </TBody>
+        <TFoot>
+          <Tr>
+            <Td>&nbsp;</Td>
+            <Td>Cell</Td>
+            <Td textAlign="center">Content</Td>
+            <Td textAlign="right">3</Td>
+          </Tr>
+        </TFoot>
+      </Table>
+    );
+  })
+  .add('Row textAlign', () => {
```

Comment:
Isn't this example still a column textAlign? 😕

Reply:
I changed this one to `Row verticalAlign` to be more clear.